config/tests/src/test/java/org/jboss/windup/config/test/metadata/MetadataAnnotationExecTest.java

```diff
@@ -0,0 +1,98 @@
+package org.jboss.windup.config.test.metadata;
+
+import java.nio.file.Paths;
+import java.util.logging.Logger;
+
+import javax.inject.Inject;
+
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.forge.arquillian.AddonDependency;
+import org.jboss.forge.arquillian.Dependencies;
+import org.jboss.forge.arquillian.archive.ForgeArchive;
+import org.jboss.forge.furnace.repositories.AddonDependencyEntry;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.windup.exec.WindupProcessor;
+import org.jboss.windup.exec.configuration.WindupConfiguration;
+import org.jboss.windup.graph.GraphContext;
+import org.jboss.windup.graph.GraphContextFactory;
+import org.jboss.windup.testutils.rulefilters.EnumerationOfRulesFilter;
+import org.jboss.windup.testutils.rulefilters.RuleFilter;
+import org.jboss.windup.util.Logging;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * Test for the Victims ruleset.
+ *
+ * @author Ondrej Zizka, ozizka at redhat.com
+ */
+@RunWith(Arquillian.class)
+public class MetadataAnnotationExecTest
+{
+    private static final Logger log = Logging.get(MetadataAnnotationExecTest.class);
+
+    @Deployment
+    @Dependencies({
+        @AddonDependency(name = "org.jboss.forge.furnace.container:cdi"),
+        @AddonDependency(name = "org.jboss.windup.utils:utils"),
+        @AddonDependency(name = "org.jboss.windup.config:windup-config"),
+        @AddonDependency(name = "org.jboss.windup.rules.apps:rules-base"),
+        @AddonDependency(name = "org.jboss.windup.rules.apps:rules-java"),
```

Comment:
This code does not appear to be using rules-base and rules-java. These can be removed.